wai-extra/Network/Wai/Test.hs

```diff
@@ -5,6 +5,7 @@ module Network.Wai.Test
     ( -- * Session
       Session
     , runSession
+    , runSessionWith, ClientState(..), initState
```

Comment:
Correct. FYI, you may be interested in: http://www.snoyman.com/blog/2017/06/how-to-send-me-a-pull-request